tools/common/Application.cs

```diff
@@ -64,7 +64,12 @@ public partial class Application
 		public bool EnableSGenConc;
 		public MarshalObjectiveCExceptionMode MarshalObjectiveCExceptions;
 		public MarshalManagedExceptionMode MarshalManagedExceptions;
-		public bool IsDefaultMarshalManagedExceptionMode;
+
+		bool is_default_marshal_managed_exception_mode;
+		public bool IsDefaultMarshalManagedExceptionMode {
+			get { return is_default_marshal_managed_exception_mode || MarshalManagedExceptions == MarshalManagedExceptionMode.Default; }
+			set { is_default_marshal_managed_exception_mode = value; }
+		}
```

Comment:
is that change needed ?
it seems new and not used elsewhere in the diff

Reply:
Yes, it's explained here: e963395a15d8ffcd6deaeac1b70a0d2f1dadcd51

It could be a different PR though (fixing the test build made a test failure show up too, but they're not directly related).

Reply:
👍 I missed there was a 2nd commit (and it's log)